sympy/physics/mechanics/_system.py

```diff
@@ -0,0 +1,937 @@
+"""Implementation of system for book-keeping all objects defining a model.
+
+Notes
+=====
+
+This module is experimental and so is named with a leading underscore to
+indicate that the API is not yet stabilized and could be subject to breaking
+changes.
+
+"""
+
+from sympy.core.backend import Basic, ImmutableMatrix as Matrix
+from sympy.core.containers import OrderedSet
+from sympy.utilities.iterables import iterable
+from sympy.physics.vector import Point, ReferenceFrame, dynamicsymbols
+from sympy.physics.mechanics.functions import _validate_coordinates, Lagrangian
+from sympy.physics.mechanics.body_base import BodyBase
+from sympy.physics.mechanics.particle import Particle
+from sympy.physics.mechanics.loads import Force, Torque, _parse_load, gravity
+from sympy.physics.mechanics.joint import Joint
+from sympy.physics.mechanics.method import _Methods
+from sympy.physics.mechanics.kane import KanesMethod
+from sympy.physics.mechanics.lagrange import LagrangesMethod
+from functools import wraps
```

Comment:
Can we follow [PEP8 import ordering](https://peps.python.org/pep-0008/#imports)? I.e. three single-line separate groups, standard library imports followed by third party imports followed by library imports, and alphabetise each group.

Reply:
Addressed in 572015c6f5163687c627a2e379d015f100a5f128